src/main/java/com/money_account_service/dtos/response/AuthorizeResponseDto.java

```diff
@@ -0,0 +1,7 @@
+package com.money_account_service.dtos.response;
+
+import lombok.Builder;
+
+@Builder
+public record AuthorizeResponseDto (String userSub, Boolean authorized){
```

Comment:
Why do you need `Boolean authorized`? If the user is not authorized, we won't return `200` response, so probably we won't even have this response dto